### PR TITLE
Portfolio - mobile menu

### PR DESCRIPTION
**Pull request by melybee-07**
we implemented the mobile menu using JavaScript to manipulate DOM elements.

### CODE REVIEW

**Comment by DeliceLydia** [comment]
## Hi @melybee-07, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that did not follow pair programming i am not able to see commit from @Bestbynature

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by huseyinbiyik** [comment]
Hi team,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

🔴 You have submitted a project, that has no Eslint linter setup. Since we start to use JavaScript for this project, now we need to set up HTML&CSS&JavaScript linters. Kindly check the project requirements.
Screenshot 2023-02-08 at 10 25 36 PM


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: styles.css [hunk 1]
```diff
@@ -779,12 +780,49 @@ main {
...
+  }
+
+  .humberger-bottun.active .ico:nth-child(3) {
+    transform: translate(-0.8rem) rotate(-45deg);
+    background-color: white;
+    margin-left: 1.5rem;
+    margin-top: -2.4rem;
+  }
+
+  .navigation-bar {
```
Comment (topeogunleye) [inline styles.css:798]: - [ ] Your Menu items should be closer to the top like seen in the Template design.
- [ ] Kindly increase the font size to match the Figma design

| Your Design  | Template Design |
| ------------- | ------------- |
| image  | image  |

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi Team 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 2]
```diff
@@ -843,9 +887,9 @@ main {
     width: 1.6rem;
   }
 
-  #sec2 {
+  /* #sec2 {
     padding: 10rem, 0;
-  }
+  } */
```
Comment (nahnahdev) [inline styles.css]: - [ ] Kindly avoid leaving comments in your code. It makes your code unclean :+1:  You can check out this [thread](https://questions.microverse.org/t/is-writing-down-comments-on-the-code-a-good-practice/687) to read more about this :+1:

File: styles.css [hunk 3]
```diff
@@ -683,7 +683,8 @@ main {
   .top-fram .bb {
     width: 19.2rem;
     height: 5.6rem;
-    padding: auto;
+
+    /* padding: auto; */
```
Comment (nahnahdev) [inline styles.css]: - [ ] Kindly avoid leaving comments in your code. It makes your code unclean :+1:  You can check out this [thread](https://questions.microverse.org/t/is-writing-down-comments-on-the-code-a-good-practice/687) to read more about this :+1:

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @melybee-07 and @Bestbynature , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.


 ## Highlights
- Requested changes solved :heavy_check_mark: 

 ## Required Changes ♻️
- Your project has some linter checks failing. Please ensure that all your linter checks are passing and you have no linter errors in your Project as per the [Project Requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md) :wink: 
- Your linter checks now
image


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED 👍 💯 

Hi @melybee-07 , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Collaborative work 👍 . Commits from both team members
:heavy_check_mark: All linter checks passing
:heavy_check_mark:  Good commit messages

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**